components/Analytics.tsx

```diff
@@ -23,21 +41,22 @@ function initializeAnalytics() {
     customBrowserType: !isMobile
       ? 'desktop'
       : 'web3' in window || 'ethereum' in window
-        ? 'mobileWeb3'
-        : 'mobileRegular',
+      ? 'mobileWeb3'
+      : 'mobileRegular',
   })
 
+  // Init brave analytics
+  initBraveAnalytics()
```

Comment:
you need to initialize on every time the URL changes? is this right?

Reply:
Usually we process UTM labels only once on page load

Reply:
I think it makes sense to do it on initial load since there will be a link somewhere with these query params and we only need to set cookies on that first load of an app once the user opens that link.